uploading_packages/remotes.rst

```diff
@@ -4,7 +4,7 @@ Remotes
 =======
 
 In the previous sections, we built several packages on our computer that were stored
-in the local cache, typically under *~/.conan/data*. Now, you might want to upload them to a Conan server
+in the local cache, typically under *~/.conan/data* or C:\conan_data. Now, you might want to upload them to a Conan server
```

Comment:
The C:\conan_data path is not typical at all. I'd say that <userhome>/.conan/data will be 99% of the cases good for the users reading the docs, why do you think it is necessary to add this?

Reply:
It's what my Windows system is using. I had to go looking for it. I installed and ran conan-win-64_1_4_5.exe

Reply:
I'm aware this is an older version of Conan -- Has this location changed in newer versions?

Reply:
I am not aware of this change but the folder is the one stated in the docs. I have Conan install via pip though, so I will have to check the installer. Maybe you have changed the conan storage path https://docs.conan.io/en/latest/reference/config_files/conan.conf.html#storage or the CONAN_USER_HOME

Reply:
Yes, ``conan_data`` does not appear in the codebase or in the docs, so the only way is that you defined the env-var at some point. Could you please check again?

Reply:
I did check again... I found this was configured independently. We can close this thread. Thanks.